examples/official-storybook/stories/addon-actions.stories.js

```diff
@@ -222,3 +222,13 @@ export const LimitActionOutput = () => {
 LimitActionOutput.story = {
   name: 'Limit Action Output',
 };
+
+export const SkippedViaDisableTrue = () => <Button onClick={action('hello-world')}>Hello World</Button>;
+
+SkippedViaDisableTrue.story = {
+  name: 'skipped via disable:true',
+
+  parameters: {
+    actions: { disable: true },
+  },
+};
```

Comment:
I can't make this work. Existing examples for disabling addons also don't work. I find the disabling centered does but not backgorunds.

Reply:
cc @tmeasday

Reply:
I suspect the issue is that `withAction.ts` doesn't leverage makeDecorator and therefore the disabling logic isn't applied. I'm not 100% sure of the path we want to take here. I'd imagine while we fix that we can make actions configurable from parameters but I'll need some guidance.

Reply:
Fixed it!

Reply:
Yeah `disable: true` is just a convention that is enforced by `makeDecorator`.